modules/geo-layers/src/h3-layers/h3-hexagon-layer.js

```diff
@@ -111,35 +111,44 @@ export default class H3HexagonLayer extends CompositeLayer {
     const {
       data,
       getHexagon,
-      updateTriggers,
-
+      wireframe,
       elevationScale,
-      extruded,
       fp64,
-
+      material,
+      extruded,
+      stroked,
+      lineWidthScale,
+      lineWidthMinPixels,
+      lineWidthMaxPixels,
       getColor,
       getElevation,
-      material
+      getLineColor,
+      getLineWidth,
+      updateTriggers
     } = this.props;
 
-    const SubLayerClass = this.getSubLayerClass('hexagon-cell-hifi', SolidPolygonLayer);
+    const SubLayerClass = this.getSubLayerClass('hexagon-cell-hifi', PolygonLayer);
 
     return new SubLayerClass(
       {
         filled: true,
         elevationScale,
         extruded,
         fp64,
-        getFillColor: getColor,
+        wireframe,
+        stroked,
+        lineWidthScale,
+        lineWidthMinPixels,
+        lineWidthMaxPixels,
+        material,
         getElevation,
-        material
+        getFillColor: getColor,
```

Comment:
If we add `getLineColor`, it's very awkward to have another prop called `getColor` instead of `getFillColor`.

To maintain backward compatibility, here's my suggestion:
- Make the `getColor` prop default to `null`
- In this line, pass `getColor || getFillColor` to the sublayer

Reply:
Noted. Resolved it.